modules/camera/camera_device.py

```diff
@@ -8,24 +8,43 @@
 import cv2
 import numpy as np
 
+try:
+    import picamera2
+except ImportError:
+    pass
```

Comment:
Set to usb camera mode

Reply:
Assuming using opencv implementation will result in usb camera mode, I'm a bit confused